atuin/src/command/client/history.rs

```diff
@@ -184,110 +184,135 @@ fn parse_fmt(format: &str) -> ParsedFmt {
 }
 
 impl Cmd {
-    pub async fn run(&self, settings: &Settings, db: &mut impl Database) -> Result<()> {
-        let context = current_context();
+    async fn handle_start(
```

Comment:
I've split `Cmd::run` into private sub-fns, because it grew larger than clippy limit :)